local-setup/simple-config.json

```diff
@@ -6,11 +6,13 @@
       "--dev",
       "-lruntime=info",
       "--ws-port",
-      "9990",
+      "9944",
```

Comment:
I think you are not using this config, but I do, and it was a pain that the config did not use the default ports.